src/rgw/rgw_op.cc

```diff
@@ -6820,7 +6802,28 @@ void RGWCompleteMultipart::complete()
     }
   }
 
+  s->object->get_attrs()[RGW_ATTR_ETAG] = target_obj->get_attrs()[RGW_ATTR_ETAG];
```

Comment:
try to remove this line and see if it still works - both in the response and the notifications